webpack.config.js

```diff
@@ -0,0 +1,57 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    library: 'AFRAME',
+    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/dist/',
+    filename: 'aframe-master.js'
+  },
+  devtool: 'source-map',
+  mode: 'development',
+  devServer: {
+    host: 'local-ip',
+    port: process.env.PORT || 9000,
+    hot: false,
+    liveReload: true,
+    static: {
+      directory: 'examples'
+    }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.INSPECTOR_VERSION': JSON.stringify(
+        process.env.INSPECTOR_VERSION
+      )
+    }),
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+      Buffer: ['buffer', 'Buffer']
+    })
```

Comment:
https://webpack.js.org/plugins/provide-plugin/

I found the solution for `process.browser` and `Buffer` here https://www.alchemy.com/blog/how-to-polyfill-node-core-modules-in-webpack-5
Note that webpack gave me this information at the beginning:
```
ERROR in ./node_modules/buffer-equal/index.js 1:13-37
Module not found: Error: Can't resolve 'buffer' in '/home/vincentfretin/workspace/aframe/node_modules/buffer-equal'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "buffer": require.resolve("buffer/") }'
	- install 'buffer'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "buffer": false }
```

but using `resolve.fallback` didn't work because `buffer-equal` doesn't even use `require('buffer')`, it just uses `Buffer` directly.